### PR TITLE
Km atac h5ad overview

### PR DESCRIPTION
**Pull request by kayleemathews**
### Description

This PR adds a Count Matrix Overview for the ATAC pipeline to the WARP website and links to the new doc in the ATAC pipeline Overview doc. It also updates pipeline version numbers for recently updated pipelines in their respective overview docs and updates links to point to master.

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: website/docs/Pipelines/ATAC/count-matrix-overview.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 2
+---
+
+# ATAC Count Matrix Overview
+
+The ATAC pipeline's default count matrix output is a h5ad file generated using [SnapATAC2](https://github.com/kaizhang/SnapATAC2) and [AnnData](https://anndata.readthedocs.io/en/latest/index.html). 
+
+The h5ad file contains unstructured metadata (`h5ad.uns`; [Table 1](#table-1-global-attributes)) as well as per-barcode quality metrics (`h5ad.obs`; [Table 2](#table-2-cell-metrics)). It also contains an equivalent gene expression barcode for each ATAC barcode. Raw fragments are stored in the `h5ad.obsm['insertion']` property of the h5ad file.
```
Comment (ekiernan) [inline website/docs/Pipelines/ATAC/count-matrix-overview.md:9]: At the end here, I would link out to snapatac2's documentation on the import_data function, which was used to generate the h5ad: https://kzhang.org/SnapATAC2/api/_autosummary/snapatac2.pp.import_data.html#snapatac2.pp.import_data

Comment (kayleemathews) [reply website/docs/Pipelines/ATAC/count-matrix-overview.md:9]: Good idea. Thanks @ekiernan!

File: website/docs/Pipelines/ATAC/count-matrix-overview.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ sidebar_position: 2
 
 The ATAC pipeline's default count matrix output is a h5ad file generated using [SnapATAC2](https://github.com/kaizhang/SnapATAC2) and [AnnData](https://anndata.readthedocs.io/en/latest/index.html).
```
[the remark this answers is not in the document]
Comment (ekiernan) [reply website/docs/Pipelines/ATAC/count-matrix-overview.md:7]:
```suggestion
The [ATAC pipeline's](README.md) default count matrix output is an h5ad file generated using [SnapATAC2](https://github.com/kaizhang/SnapATAC2) and [AnnData](https://anndata.readthedocs.io/en/latest/index.html).

File: website/docs/Pipelines/ATAC/count-matrix-overview.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+The h5ad file does not contain per-gene metrics, meaning the variables/features data frame (`h5ad.var`) is empty.
+
+
+## Table 1. Global attributes
+
+The global attributes (unstuctured metadata) in the h5ad apply to the whole file, not any specific part. 
+
+| Attribute | Program | Details |
+| --- | --- | --- |
+| `reference_sequences` | [SnapATAC2](https://github.com/kaizhang/SnapATAC2) | Data frame containing the chromosome sizes for the genome build (i.e., hg38); created using the `chrom_sizes` pipeline input. |
```
Comment (ekiernan) [inline website/docs/Pipelines/ATAC/count-matrix-overview.md:20]:
```suggestion
| `reference_sequences` | [SnapATAC2](https://github.com/kaizhang/SnapATAC2) | Data frame containing the chromosome sizes for the genome build (i.e., hg38); created using the [`chrom_sizes` pipeline input](README.md). |
```

File: website/docs/Pipelines/ATAC/count-matrix-overview.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## Table 2. Cell metrics
+
+| Cell Metrics | Program | Details |
+|---|---|--------------------|
+| `tsse` | [SnapATAC2](https://github.com/kaizhang/SnapATAC2) | Transcription start site enrichment (TSSe) score; lower scores suggest poor data quality. Learn more about TSSe in the [Definitions section](#definitions) below. |
+| `n_fragment` | [SnapATAC2](https://github.com/kaizhang/SnapATAC2) | Number of unique fragments corresponding to the ATAC cell barcode. Fragments are stored in the `h5ad.obsm` property of the output h5ad file. Learn more about cell barcodes and fragments in the [Definitions section](#definitions) below. |
+| `frac_dup` | [SnapATAC2](https://github.com/kaizhang/SnapATAC2) | Fraction of reads associated with the cell barcode that are duplicates. |
+| `frac_mito` | [SnapATAC2](https://github.com/kaizhang/SnapATAC2) | Fraction of reads associated with the cell barcode that are mitochondrial. |
+| `gex_barcodes` | [AnnData](https://anndata.readthedocs.io/en/latest/index.html) | Gene expression barcode associated with each ATAC cell barcode. |
```
Comment (ekiernan) [inline website/docs/Pipelines/ATAC/count-matrix-overview.md:31]:
```suggesting
| `gex_barcodes` | [AnnData](https://anndata.readthedocs.io/en/latest/index.html) | Gene expression barcode associated with each ATAC cell barcode. This column is only produced when ATAC is run as part of the Multiome pipeline. |
```

**Review by ekiernan — APPROVED** [review]
One more small suggestion, but otherwise it LGTM! Thank you so much for writing this up.

**Review by kevinpalis — APPROVED** [review]
Looks great!

File: website/docs/Pipelines/ATAC/count-matrix-overview.md (hunk 4 above)
Comment (kayleemathews) [reply website/docs/Pipelines/ATAC/count-matrix-overview.md:31]: Good add. Thanks Liz!

**Pull request merged**